app/pods/publications/publication/controller.js

```diff
@@ -170,6 +170,13 @@ export default class PublicationController extends Controller {
 
   @action
   allowedTranslationDate(date) {
+    // If translateBefore has expired, show that date (input is empty without this)
+    const translateBefore = this.model.publicationFlow.get('translateBefore');
+    if (translateBefore && moment(translateBefore).isBefore(moment())) {
+      if (moment(date).isSame(translateBefore)) {
+        return true;
+      }
+    }
```

Comment:
We checken eerst of translateBefore ouder is dan vandaag
Indien ja dan laten we die waarde toe + de range tussen vandaag en publishBefore.
Zonder dit was het input veld van translateBefore leeg als de datum vervallen was